lang/en.json

```
@@ -563,7 +563,7 @@
   "CP.Contributions.ConnectedCollective": "{n, plural, one {This Collective is} other {These Collectives are}} connected to us",
   "CP.Contributions.PartOfOrg": "{n, plural, one {This Collective is} other {These Collectives are}} part of our Organization",
   "create.collective.placeholder": "Making the world a better place",
-  "create.pledge.stats": "by {orgCount, plural, =0 {} one {# sponsor} other {# sponsors}} {both, plural, =0 {} one { and }} {userCount, plural, =0 {} one {# backer } other {# backers }}",
+  "create.pledge.stats": "by {orgCount, plural, =0 {} one {# sponsor} other {# sponsors}} {both, plural, =0 {} one { and } other {}} {userCount, plural, =0 {} one {# backer } other {# backers }}",
```

Maybe we can simplify? (not sure if this works)

```suggestion
  "create.pledge.stats": "by {orgCount, plural, =0 {} one {# sponsor} other {# sponsors}} {both, plural, =0 {} other { and }} {userCount, plural, =0 {} one {# backer } other {# backers }}",
```

---

Yes, this works. It think the other needs to be there; that's the only condition. 😄